### PR TITLE
Update AM and AME-LFTE footer block to go to my.auntminnie preference center

### PR DESCRIPTION
**Pull request by jwade1327**
With subID passed through:
Screen Shot 2024-01-26 at 2 14 32 PM
Screen Shot 2024-01-26 at 2 15 38 PM

Screen Shot 2024-01-26 at 10 59 28 AM
Screen Shot 2024-01-26 at 11 05 23 AM
Screen Shot 2024-01-26 at 11 23 16 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Change the AuntMinnie Breaking News template footer](https://trello.com/c/SiuK6wYt/514-change-the-auntminnie-breaking-news-template-footer)

**Comment by jwade1327** [comment]
[Change the AuntMinnie Case of the Week template footer](https://trello.com/c/k7PfZmW7/515-change-the-auntminnie-case-of-the-week-template-footer)

**Comment by jwade1327** [comment]
[Change the AuntMinnie Community Insider template footer](https://trello.com/c/pOtGOHNg/513-change-the-auntminnie-community-insider-template-footer)

**Comment by jwade1327** [comment]
[Change the AuntMinnieEurope LFTE template footer](https://trello.com/c/ywSVgINt/517-change-the-auntminnieeurope-lfte-template-footer)

File: packages/common/components/blocks/am-footer.marko [hunk 1]
```diff
@@ -19,7 +20,7 @@ $ const preferenceCenterUrl = buildLinkUrl(
   {
     email: '{{${email_address} | url_encode }}',
     id: '{{${user_id}}}',
-    subtitle: 'AuntMinnie Daily Newsletter',
+    subtitle: `${subtitle}`,
```
Comment (Shinsina) [inline packages/common/components/blocks/am-footer.marko:23]: Believe this can be simplified to just `subtitle,`?

**Review by Shinsina — CHANGES_REQUESTED** [review]

**Review by Shinsina — APPROVED** [review]

**Pull request merged**